lib/src/rewrite.rs

```diff
@@ -524,8 +524,6 @@ impl<'settings, 'repo> DescendantRebaser<'settings, 'repo> {
         Ok(())
```

Comment:
Typo in commit description: address -> addressed (or "we will address"?)

Reply:
Fixed, thanks.